styles/style.css

```diff
@@ -0,0 +1,333 @@
+nav {
+  opacity: 1;
+  background-color:#F4F4F4;
+  position: sticky;
```

Comment:
Love the use of this! Very effective for scrolling, especially with that bottom border on the nav.